usr/share/rear/verify/CDM/default/410_use_replica_cdm_cluster_cert.sh

```diff
@@ -32,7 +32,7 @@ StopIfError "Could not download https://${CDM_CLUSTER_IP}/connector/${CDM_SUNOS_
 /usr/bin/tar $v -xzf  $CDM_SUNOS_TAR
 StopIfError "Could not extract $CDM_SUNOS_TAR"
 
-CDM_CERT_FILE=$(find $v ./ -name "rubrik.crt")
+CDM_CERT_FILE=$(find ./ -name "rubrik.crt")
```

Comment:
This change is already merged via
https://github.com/rear/rear/pull/2267
so that this change should be removed from this pull request
to keep this pull request clean for only the one single issue
that it actually is about i.e. to only "fix for CentOS 7.7 and 8.0
where certain programs (e.g. dhclient) require additional libraries".